packages/contractkit/src/wrappers/LockedGold.ts

```diff
@@ -13,6 +18,10 @@ import {
   valueToString,
 } from '../wrappers/BaseWrapper'
 
+type AddressListItem = ALI<BigNumber>
+// BigNumber comparator
+const bnc: Comparator<BigNumber> = (a: BigNumber, b: BigNumber) => a.lt(b)
```

Comment:
I would add a more readable name, or you could define here a wrapper function of the `linkedListChanges` that already uses this comparator, and use that instead

Reply:
done